target/linux/rtl838x/config-5.4

```diff
@@ -0,0 +1,226 @@
+CONFIG_ARCH_32BIT_OFF_T=y
+CONFIG_ARCH_CLOCKSOURCE_DATA=y
```

Comment:
Just pulled the PR and ran "make kernel_oldconfig" on it, changed quite a bit. Didn't touch CONFIG_SWCONFIG yet ...